google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/CloudLibraryProjectStateTest.java

```diff
@@ -117,9 +117,12 @@ public void getCloudLibraries_withMalformedPom_isEmpty() {
                   .runWriteAction(
                       () -> {
                         try {
-                          mavenProject.getFile().setBinaryContent("not a valid pom.xml".getBytes());
+                          mavenProject
+                              .getFile()
+                              .setBinaryContent(
+                                  "not a valid pom.xml".getBytes(Charset.defaultCharset()));
                         } catch (IOException e) {
-                          fail();
+                          throw new AssertionError(e);
```

Comment:
It'd be good to add a message to the assertion error just to emphasize what is failing about the test.  It'll also make it clear that AssertionError is thrown by the test code.

Reply:
btw, `fail` internally throws an `AssertionError`. So if this update silences the warning then its just because the analyzer isn't smart enough.

Reply:
oh nm. its about the stacktrace being lost. lgtm (after @patflynn comment)